### ISSUE TITLE
Ignored PDFs are not removed from the PDF list (drop down filter)

### ISSUE THREAD

**Issue by feldegast** [openaudible/openaudible#1130]
setting a PDF does not move it out of the list of PDFs shown when PDF is selected from the top filter (dropdown list) or indicate the pdf is ignored in some way.... the status text still says ready to download....

I realise this is not a core feature just noticed it doesn't follow the expected behaviour used in the rest of the app....

**Comment by openaudible** [comment]
Yeah.. will fix for next beta. Counts in the Library status will still show counts from ignored books however. 
Thanks and closing to reduce clutter.
